code/datums/keybinding/xeno.dm

```diff
@@ -335,6 +335,13 @@
 	description = ""
 	keybind_signal = COMSIG_XENOABILITY_PSYCHIC_FLING
 
+/datum/keybinding/xeno/unrelenting_force
+	key = "Unbound"
+	name = "unrelenting_force"
+	full_name = "Unrelenting Force"
+	description = ""
+	keybind_signal = COMSIG_XENOABILITY_PSYCHIC_FLING
```

Comment:
It should have its own signal.

Reply:
i was pretty tired when i was adding the keybindings, must of missed that